apps/condo/domains/user/components/auth/InputPhoneForm.tsx

```diff
@@ -70,8 +77,10 @@ export const InputPhoneForm: React.FC<IInputPhoneFormProps> = ({ onFinish })=> {
             onCompleted: (data) => {
                 const { data: { result: { token } } } = data
                 setToken(token)
-                Router.push(`/auth/register?token=${token}`)
-                onFinish()
+                router.push(`/auth/confirm?token=${token}`)
```

Comment:
Extract it to effect, with dependency on `token`.
This will help to remove duplicated code above.